src/modules/sub-items/sub.items.module.js

```diff
@@ -328,7 +328,7 @@ SubItemsModule.propTypes = {
     updateLocationPriority: PropTypes.func,
     activeView: PropTypes.string,
     extraActions: PropTypes.arrayOf(PropTypes.shape({
-        component: PropTypes.element,
+        component: PropTypes.func,
```

Comment:
It's not a function it's a React class reference.
Maybe you should implement custom checking? https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
I was thinking about it, I will investigate it further and I will give feedback. :)

Reply:
I checked it and options depend on what we assume about component. If it is:
1. _class component_ - we can check if it inherits from React.Component
2. _functional component_ or _class component_ - we can only check if it is a function (see: https://github.com/facebook/react/issues/9186#issuecomment-289695205)
3. HTML or _2._ - we can check if it is a _function_ or a _string_

Reply:
@sunpietro What do you think?

Reply:
Let it be of `function` type then